x/gov/context.go

```diff
@@ -0,0 +1,56 @@
+package gov
+
+import (
+	"context"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x"
+)
+
+type contextKey int // local to the gov module
+
+const (
+	contextKeyGov contextKey = iota
+)
+
+// withElectionSuccess is a private method, as only this module
+// can add a gov signer
```

Comment:
I can see this is a private method because of the lower case. Only this module can add a gov signer because this is a private method :spaghetti: 
This function is straighforward and I can read **what** it is doing. It would be much better to instead of this comment explain **why** only this module can add a gov signer.